go.mod

```diff
@@ -1,3 +1,16 @@
 module github.com/sigstore/rekor-tiles
 
 go 1.24.0
+
+require (
+	github.com/spf13/cobra v1.9.1
+	sigs.k8s.io/release-utils v0.11.0
+)
+
+require (
+	github.com/common-nighthawk/go-figure v0.0.0-20210622060536-734e95fb86be // indirect
```

Comment:
Just noticed, all these indirect deps except github.com/spf13/pflag are unmaintained.

Reply:
I think these come from cobra too. We could just use the std library for Go flags, and see about pulling in the logic to get app version from k8s-release-utils/